templates/00-master.yaml

```diff
@@ -443,13 +392,13 @@ Parameters:
   WebAsgMax:
     AllowedPattern: ^((?!0$)[1-2]?[0-9]|30)$
     ConstraintDescription: Must be a number between 1 and 30.
-    Default: 1
+    Default: 2
```

Comment:
@sharepointoscar Does this value need to be set to 2? Or is this for testing?

Reply:
This is just for testing and you can change the value via the `.taskcat.yaml` which I provided with all parameters needed.